Assets/_Project/Scripts/UserInterface/Components/List/ListElement.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using JetBrains.Annotations;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Components.List
+{
+    public class ListElement
+    {
+        public string ButtonText { get; }
```

Comment:
Shouldn't these fields be private fields with public accessors? For example:

```
private string _bottomText;

public string BottomTexxt
{
get{ return _bottomText;}
}
```

Reply:
This is a read-only property syntax, backing field is created automatically, so there is no need for that.

Reply:
ok